src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -63,6 +72,7 @@ Drawer {
       camera.stop()
       photoPreview.visible = false
     }
+    photoPreview.source = ""
```

Comment:
I don't fully understand why we need to set photo preview source to empty string here - especially why both when making the item visible and also when hiding it?